packages/document-service-api/src/controllers/documents.js

```diff
@@ -88,9 +88,11 @@ const serveDocumentById = async (req, res) => {
       return;
     }
 
+    const fileName = metadata.name;
     const mimeType = metadata.mime_type;
     req.log.info({ mimeType }, 'Displaying file');
     res.set('content-type', mimeType);
+    res.set('x-original-file-name', fileName);
```

Comment:
Why do we need to return the filename here?

Reply:
it can be useful to know the name of the file so that that name can be used to naming the file on download @shepda

Reply:
We already have the name of the file in the appeal data in the UI so couldn't we use that for the name of the downloaded file?